solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-9.adoc

```diff
@@ -68,6 +68,11 @@ Due to changes in Lucene 9, that isn't possible any more.
 === Querying and Indexing
 * Added Lucene91HnswVectorsFormat codec for DenseVectorField. In order to use the new codec, reindex is necessary.
 
+=== SolrJ
+SolrJ is beginning to be split up.  If you use ZooKeeper coordinates to create a `CloudSolrClient`, you will need to add a dependency on `solrj-zookeeper`.
```

Comment:
As I understand, this will be a transitive dependency in Solr 9.1, so this text should say that no changes are needed as long as you enable automatic dependency resolution in maven/gradle.

Reply:
Sure.
> SolrJ is beginning to be split up.  If you use ZooKeeper coordinates to create a `CloudSolrClient`, you will need to add a dependency on `solrj-zookeeper`.  If you use SolrJ's Maven POM to depend on SolrJ, then this should happen automatically through transitive resolution.